services/alertmanager/alertmanager.go

```diff
@@ -181,6 +207,65 @@ func (svc *Service) updateConfiguration(ctx context.Context) {
 	svc.l.Infof("%s created", alertmanagerConfigPath)
 }
 
+// collectRuleTemplates collects IA rule templates from various sources like
+// templates shipped with PMM and defined by the users.
+func (svc *Service) collectRuleTemplates() {
+	shippedFilePaths, err := filepath.Glob(svc.shippedRuleTemplatePath)
```

Comment:
Looks like proper size here should be `len(svc.shippedRuleTemplatePath + svc.userDefinedRuleTemplatePath)`

Reply:
we don't need that anymore, now we use maps

Reply:
Actually, map also has a size)